src/types.d.ts

```diff
@@ -17,7 +17,7 @@ export type Rule = {
 }
 
 export type LooseObject = {
-	/* eslint-disable @typescript-eslint/no-explicit-any */
+	/* eslint-disable-next-line @typescript-eslint/no-explicit-any */
```

Comment:
We should always use `eslint-disable-next-line` instead of `eslint-disable`. The latter seems to disable for the entire file, instead of just the function.
I'm not sure why that is. My suspicion is it's something related to the parser.